.github/pull_request_template.md

```diff
@@ -1,9 +1,11 @@
-<!-- Add a small description here of the changes you added -->
+## Edited
+<!-- Add a brief description here -->
 
-- Added/Edited note ...
+## Added
+<!-- Add a brief description here-->
 
 ## Checklist
-
-- [ ] I have renamed all attached images with descriptive file names (or I did not include any images)
-- [ ] Before creating a new note, I searched the vault (or I only edited existing notes)
-- [ ] (Optional) In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).
+- [ ] before creating a new note, I searched the vault
+- [ ] new files have the `.md` extension
```

Comment:
Pedantry, but I think this should be "new notes..." as new images will have a different extension